force-app/emails/lwc/emailDeliveryReport/__tests__/emailDeliveryReport.test.js

```diff
@@ -0,0 +1,25 @@
+import { createElement } from 'lwc';
+import EmailDeliveryReport from 'c/emailDeliveryReport';
+
+describe('c-email-delivery-report', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('TODO: test case generated by CLI command, please fill in test logic', () => {
```

Comment:
There are still a few TODOs in here. Did you want to check those in or were you planning to complete them before merge?

Reply:
Writing tests is in the backlog as #67